monitoring/monitors/viya/serviceMonitor-sas-pushgateway.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 spec:
   selector:
     matchLabels:
-      app: prometheus-pushgateway
+      app.kubernetes.io/name: prometheus-pushgateway
```

Comment:
Should we match on either one of these labels?  Do we need to worry about sites with multiple versions of SAS Viya and/or multiple versions of SAS Viya Monitoring?